src/util/isEventEmitter.ts

```diff
@@ -0,0 +1,15 @@
+export type EventEmmitterLike1 = {
```

Comment:
They'll need to be exported otherwise you'll raise a compiler exception.  What we don't have to do is export the from `Observable` and/or `KitchenSink`, so they'll be hard to get at.

Also since they're exported types unless someone tries to explicitly import this type, into an environment where this is imported as an ambient type (anything with `node.d.ts` really), there shouldn't not be any problems.  And long as the type shape matches, the ambient type would fit inside the exported type.

We probably don't need a separate `isEventEmitter` file though do we? This could just live as a private method in `fromEvent.ts`